.teamcity/Gradle_Check/configurations/Gradleception.kt

```diff
@@ -13,7 +13,7 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(mod
     description = "Builds Gradle with the version of Gradle which is currently under development (twice)"
 
     params {
-        param("env.JAVA_HOME", "%linux.java8.oracle.64bit%")
+        param("env.JAVA_HOME", "%linux.java9.oracle.64bit%")
```

Comment:
I think we should extract this placeholders to some constants.

Reply:
Absolutely. Already done.